topo_processor/metadata/collection_test.py

```diff
@@ -0,0 +1,11 @@
+from topo_processor.metadata import Collection, DataType
+
+
+def test_collection_has_title():
```

Comment:
I am not sure how important these tests are, you are only validating the constructor actually assigns something.

if anything you should have one test that does all the validation of the constructor